packages/pipeline-editor/src/common-canvas.d.ts

```diff
@@ -596,6 +596,7 @@ declare module "@elyra/canvas" {
     app_data?: {
       // NOTE: elyra specific.
       version: number;
+      pipeline_properties: any;
```

Comment:
Can we actually change this to just `properties`? `pipeline_properties` feels a bit redundant since it's the pipeline level app_data. I think we should also wrap the node properties inside of a `properties` object when we do the pipeline version bump